tests/components/brother/test_config_flow.py

```diff
@@ -40,8 +40,8 @@ async def test_create_entry_with_hostname(hass):
         assert result["data"][CONF_TYPE] == CONFIG[CONF_TYPE]
 
 
-async def test_create_entry_with_ip_address(hass):
-    """Test that the user step works with printer IP address."""
+async def test_create_entry_with_ipv4_address(hass):
```

Comment:
@KapJI I've noticed that my vscode editor is now complaining about non typed tests for strictly typed integrations like zwave_js. Can we tweak the mypy settings to avoid that?

Reply:
It seems we have no special settings for tests. I think we should ignore the tests by default.

Reply:
My guess is that before all mypy errors were ignored, now we have default config with strict checks.
Although we don't run mypy on tests, if you run it manually it will tell about all these errors. Vscode probably does this under the hood.

I think we can disable strict checks for tests in mypy config. Some errors will remain but it should be fine.

Or we can ignore all errors in tests but then it will be quite hard to run mypy on tests if someone wants to do this manually.

Reply:
The first option may be ok. Personally I think typing our tests is not very useful as the test functions are run individually and there is no return value for them. Only the pytest fixtures could be useful to type and the parameters that they form in the test functions. But it's work for little gain.

Reply:
I think checks may be useful there not to check tests themself but to check how they use typed code. Basic checks for annotated parts should do this work just fine.